src/packs/cli.rs

```diff
@@ -186,6 +189,13 @@ pub fn run() -> anyhow::Result<()> {
 
     install_logger(args.debug);
 
+    match args.command {
```

Comment:
Any reason not to have `packs::init` be called in the case statement below?

Reply:
This was because `configuration = packs::configuration::get(&absolute_root)` blows up if you set the root path to and configuration-less directory (as you would in the case of `init`ing in a new project) - `No root pack found. First double check a root pack exists`. I was not sure how to restructure everything to allow for this fact (ie that before an `init` command there isnt a configuration) ... so I put the processing of the command before the configuration load... hacky but not well versed enough in Rust to think of a better solution!